ShelterTechSF/sheltertech.org
# Enable prismic previews

## Description

This PR enables Prismic previews! Right now, only the Home Page is previewable, but future pages will be easy to enable -- we just have to add `_previewable` to the GraphQL query and wrap the exported page with `withPrismicPreview()`.

Note that non-page things cannot be previewed with this setup -- so previewing the "Footer", for example, won't work.

Note also that we need to add a schema for _every_ Prismic type in `gatsby-config.js`, or previews will break...even if we deleted the type, we at least need to assign it an empty object `{}`. If you start getting errors when trying to preview something in Prismic, this is the first place you should look -- make sure everything has a schema, and all of the schemas are up to date.

## Testing Steps

- Check out this branch and run it locally
- Visit the Prismic website
- Under "Documents" click on "Home Page"
- Confirm that it has visible unpublished changes (currently the header title has some exclamation points)
- Click the preview "eye" icon in the upper right and select "Local" ("Production" won't work until this is merged)
- A new tab should open with the preview, showing the unpublished changes
